public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/CloudDataSourceSelector.tsx

```diff
@@ -4,24 +4,14 @@ import { useFormContext } from 'react-hook-form';
 
 import { DataSourceInstanceSettings, GrafanaTheme2 } from '@grafana/data';
 import { Field, InputControl, useStyles2 } from '@grafana/ui';
-import { contextSrv } from 'app/core/services/context_srv';
-import { AccessControlAction } from 'app/types';
 
 import { RuleFormType, RuleFormValues } from '../../../types/rule-form';
 import { CloudRulesSourcePicker } from '../CloudRulesSourcePicker';
-import { RuleTypePicker } from '../rule-types/RuleTypePicker';
-
-interface Props {
-  editingExistingRule: boolean;
-}
-
-export const AlertType = ({ editingExistingRule }: Props) => {
-  const { enabledRuleTypes, defaultRuleType } = getAvailableRuleTypes();
 
+export const CloudDataSourceSelector = ({ dataSourceSelected }: { dataSourceSelected: string | null }) => {
```

Comment:
this component was originally the AlertType, including the type picker and the cloud data source selector.